samples/create_project.py

```diff
@@ -45,7 +45,8 @@ def main():
     logging.basicConfig(level=logging_level)
 
     tableau_auth = TSC.PersonalAccessTokenAuth(args.token_name, args.token_value, site_id=args.site)
-    server = TSC.Server(args.server, use_server_version=True)
+    server = TSC.Server(args.server)
+    server.use_server_version()
```

Comment:
why change this?

Reply:
imperfect undo. But it does make it easier to insert the ssl change for local testing :)